qsirecon/interfaces/dipy.py

```diff
@@ -589,3 +607,68 @@ def _run_interface(self, runtime):
             self._results[metric] = out_name
 
         return runtime
+
+
+class _KurtosisReconstructionMicrostructureInputSpec(DipyReconInputSpec):
+    kurtosis_clip_min = traits.Float(-0.42857142857142855, usedefault=True)
+    kurtosis_clip_max = traits.Float(10.0, usedefault=True)
+
+
+class _KurtosisReconstructionMicrostructureOutputSpec(DipyReconOutputSpec):
+    ad = File()
+    ade = File()
+    awf = File()
+    axonald = File()
+    kfa = File()
+    md = File()
+    rd = File()
+    rde = File()
+    tortuosity = File()
+    trace = File()
+
+
+class KurtosisReconstructionMicrostructure(DipyReconInterface):
+    input_spec = _KurtosisReconstructionMicrostructureInputSpec
+    output_spec = _KurtosisReconstructionMicrostructureOutputSpec
+
+    def _run_interface(self, runtime):
+        gtab = self._get_gtab()
```

Comment:
Probably for a different PR, but I'm realizing that we can use Large and Small deltas in the dipy gtab just like we use them in tortoise. I don't think they make a difference in the DKI fit, but for others it probably will